src/components/terms-and-conditions.tsx

```diff
@@ -0,0 +1,151 @@
+            <div className="flex min-h-full items-center justify-center p-4 text-center">
+              <Transition.Child
+                as={Fragment}
+                enter="ease-out duration-300"
+                enterFrom="opacity-0 scale-95"
+                enterTo="opacity-100 scale-100"
+                leave="ease-in duration-200"
+                leaveFrom="opacity-100 scale-100"
+                leaveTo="opacity-0 scale-95"
+              >
+                <Dialog.Panel className="w-auto max-w-5xl transform overflow-hidden rounded-2xl bg-white p-6 text-left align-middle shadow-xl transition-all">
+                  <Dialog.Title
+                    as="h3"
+                    className="border-b p-5 text-center text-2xl font-medium leading-6 text-gray-900"
+                  >
+                    House Rules
+                  </Dialog.Title>
+                  <div className="mt-2">
+                    <p className="text-m text-gray-500">
+                      <p className="m-6 gap-y-4 pl-10 pr-10 text-center text-red-600">
+                        *Note: Please check the checkbox to acknowledge that you
+                        have read and accept our House Rules.*{" "}
+                      </p>
+
+                      <ol className="tlex grid list-decimal justify-center gap-y-4 pl-10 pr-10">
+                        <li>
+                          Workshop participants carry out the repair themselves
+                          whenever possible, and repair volunteers are on site
+                          to help or provide advice, if necessary.
+                        </li>
+                        <li>
+                          The work carried out in Repair Lab by repair
+                          volunteers is performed free of charge on a voluntary
+                          basis.
+                        </li>
+                        <li>
+                          The fact that the repairs are being performed by
+                          unpaid volunteers reflects the allocation of risks and
+                          limitation of liability: neither the organisers of
+                          Repair Lab nor the repair volunteers are liable:
+                          <ol className="list-disc">
+                            <li>
+                              for any loss that may result from advice or
+                              instructions concerning repairs,
+                            </li>
+                            <li>
+                              for the loss of items handed over for repair,
+                            </li>
+                            <li>for indirect or consequential loss,</li>
+                            <li>
+                              for any other kind of loss resulting from work
+                              performed at Repair Lab.
+                            </li>
+                          </ol>
+                        </li>
```

Comment:
The points under `Point 3` are not indented.
<img width="1113" alt="Screenshot 2023-07-12 at 5 03 04 pm" src="https://github.com/codersforcauses/repair-labs/assets/62389033/80787b98-fe94-4e73-8688-2ee66862bd39">